requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -22,50 +22,52 @@ This campaign triggers when a user completes the `user signed up` event for the
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Product Teams email.
+            1. If `Yes`, send NEW email promoting the user interview survey template
+            2. If `No`, send Advice for Product Teams email.
         3. Wait 12 hours.
     4. If `marketing` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Marketing Teams email.
+            1. If `Yes`, send NEW email promoting the dashboard templates. 
+            2. If `No`, send Advice for Marketing Teams email.
         3. Wait 12 hours.
     5. If `sales` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Sales Teams email.
+            1. If `Yes`, send NEW email promoting the tutorials section.
+            2. If `No`, send Advice for Sales Teams email.
         3. Wait 12 hours.
     6. If not `founder` `engineering` `product` `marketing` or `sales` then wait 24 hours
 4. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
 5. Send AARRR intro article email
 6. Wait 1 week.
 7. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
 8. Check if the user is in the `Subscribers to Session Replays` segment
-    1. If `Yes`, do nothing. (See Session replay onboarding flow)
-    2. If `No`, assign to a random cohort branch.
-        1. 50% receive Session replay upsell email
-        2. 50% recieve Experiment: Personal Invite email
-9. Wait 1 week.
+    1. If `Yes`, check if the user is in the `Subscribers to Feature Flags` segment
+        1. If `Yes`, if the user is in the `Subscribers to Surveys` segment
+            1. If `Yes`, send [New email about using templates]
+            2. If `No`, send Survey Upsell email.
+        2. If `No`, send Feature Flag Upsell email
+    2. If `No`, send Replay Upsell email. 
+9. Wait 1 week. 
 10. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
 11. Check if the user is in the `Subscribers to Feature Flags` segment
-    1. If `Yes`, do nothing. (See Feature flag onboarding flow)
-    2. If `No`, assign to random cohort branch. 
-        1. 50% receive Feature Flag Upsell email 1
-        2. 50% recieve Feature Flag Upsell email 2
-12. Wait 5 days. 
+    1. If `Yes`, check if the user is in the `Subscribers to Surveys` segment
+        1. If `Yes`, [send new email about Toolbar]
+        2. If `No`, send Surveys Upsell email
+    2. If `No`, send Feature Flag Upsell email
+12. Wait 1 week. 
 13. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
 14. Check if the user is in the `Subscribers to Surveys` segment
-    1. If `Yes`, do nothing. (See Survey onboarding flow)
+    1. If `Yes`, [send new email about notebooks]
```

Comment:
<img width="1026" alt="Screenshot 2023-11-09 at 14 35 37" src="https://github.com/PostHog/meta/assets/84011561/3af3bb6d-550d-42d8-9062-13fd68636655">

Reply:
instead of "multiple insights or sessions" maybe we should restructure to say "multiple insights and sessions".

Reply:
<img width="822" alt="Screenshot 2023-11-09 at 16 33 26" src="https://github.com/PostHog/meta/assets/84011561/340a9a41-935f-49e9-b6de-3b90f1351146">